nix/build/ocaml-overlay.nix

```diff
@@ -1,16 +1,23 @@
-# SPDX-FileCopyrightText: 2021 Oxhead Alpha
+# SPDX-FileCopyrightText: 2021-2022 Oxhead Alpha
 # SPDX-License-Identifier: LicenseRef-MIT-OA
 
 { sources ? import ../nix/sources.nix
 , protocols ? builtins.fromJSON (builtins.readFile ../../protocols.json)
-, hacks ? import ./hacks.nix
-, pkgs ? import sources.nixpkgs { }, opam-nix ? import sources.opam-nix pkgs }:
-self: super: {
-  ocamlPackages = self.ocaml-ng.ocamlPackages_4_12.overrideScope'
-    (builtins.foldl' self.lib.composeExtensions (_: _: { }) [
-      (opam-nix.traverseOPAMRepo' sources.opam-repository)
-      (oself: osuper: { index-super = osuper.index.versions."1.2.0"; })
-      (opam-nix.callOPAMPackage sources.tezos)
+, hacks ? import ./hacks.nix, zcash-overlay ? import ./zcash-overlay.nix
```

Comment:
Maybe I'm missing something, but why is `hacks` still needed? :thinking:

Reply:
There are a couple of Octez-specific workarounds needed e.g. for `tezos-admin-client` binary (since it's the part of the `tezos-client` opam package)